OpenSim/ApplicationPlugins/CreateCommsManager/CreateCommsManagerPlugin.cs

```diff
@@ -160,7 +160,7 @@ protected virtual void InitializeGridServices(LibraryRootFolder libraryRootFolde
 
             m_httpServer.AddStreamHandler(new OpenSim.SimStatusHandler());
             m_httpServer.AddStreamHandler(new OpenSim.XSimStatusHandler(m_openSim));
-            if (m_openSim.userStatsURI != String.Empty )
+            if (!String.IsNullOrEmpty(m_openSim.userStatsURI))
```

Comment:
Not a semantic change: both assignments to `userStatsURI` in `OpenSimBase` are to a non-null value.